models/test/mapper/TestPreviewMapper.php

```diff
@@ -230,11 +231,16 @@ private function getService(): Service
 
     /**
      * @param $categories
-     *
+     * @param AssessmentItemRef|ExtendedAssessmentItemRef $itemRef
      * @return bool
      */
-    private function isItemInformational($categories): bool
+    private function isItemInformational($categories, $itemRef): bool
     {
-        return in_array('x-tao-itemusage-informational', $categories, true);
+        $additionalCheck = false;
+
+        if (method_exists($itemRef, 'getResponseDeclarations') && !count($itemRef->getResponseDeclarations())) {
```

Comment:
Can we use `empty` here?
```
if (method_exists($itemRef, 'getResponseDeclarations') && !count($itemRef->getResponseDeclarations())) {
```
========>
```
if (method_exists($itemRef, 'getResponseDeclarations') && empty($itemRef->getResponseDeclarations())) {
```

Reply:
Just by curiosity, why use empty instead of count? empty is less precise than count and doesn't say what we're testing.
All the more, if you want to test that `$itemRef->getResponseDeclarations()` is an empty array, why not doing it explicitly with
```
$itemRef->getResponseDeclarations() === []
```
?